automation/DeployToHPVS.bashlib

```diff
@@ -498,8 +504,12 @@ Login to your IBM Cloud account and navigate to Manage > Access (IAM) > API keys
     #--------------------------------------------------------------------------#
     # Config variable: registryURL                                             #
     #--------------------------------------------------------------------------#
-    prompt_user "Enter URL of container registry. For example, us.icr.io for IBM Cloud Container Registry
-For more information about creating a container registry in IBM Cloud see here https://cloud.ibm.com/apidocs/container-registry" false "docker.io"
+    prompt_user "
+===============================================================
+Enter URL of container registry. For example, us.icr.io for IBM Cloud Container Registry
+For more information about creating a container registry in IBM Cloud see here:
```

Comment:
this one feels a little strange cause there's no prompt line at the end, but then I went back and checked and the previous one didn't have it either.  not sure what I would change though.  I think its ok for now and we can think of how to improve it over time